OpenRA.Game/VoiceExts.cs

```diff
@@ -49,7 +49,7 @@ public static bool HasVoice(this Actor self, string voice)
 			return self.TraitsImplementing<IVoiced>().Any(x => x.HasVoice(self, voice));
 		}
 
-		public static void PlayVoiceForOrders(this World w, Order[] orders)
+		public static void PlayVoiceForOrders(Order[] orders)
```

Comment:
This means this is no longer an extension method and living in `VoiceExts` seems weird.

Should we rename it? Or maybe make it `this Order[] orders` so we can use it as an extension on an array of orders?